README.md

```diff
@@ -1 +1,42 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
\ No newline at end of file
+# .env:
+
+database name, host, port, passsword and user to connect 
+
+- PG_HOST: postgresql host
+- PG_USER: postgresql username
+- PG_PASSWORD: postgresql password
+- PG_PORT: postgresql port
+- PG_DBNAME: postgresql dbname
+
+# docker for postgresql and installation of some modules:
+
+- chmod +x requirements/init_script.sh
+- ./requirements/init_script.sh
+
+# run:
+- python3 main.py
+
+# POST, PUT, DELETE examples
+
+POST example: 
+
+- in HEADERS you have "Authorization" as key and "admin {YOUR_TOKEN}" as value
```

Comment:
how do i know the deafault token that you create? you don't even know it yourself, because u generate while initializing your database.
and why admin then?
you need to add default token manually in .ddl and use it in your readme examples.
also - where do i need to send my requests? what address? what page? i guess its 127.0.0.1:8001/main, but im just guessing here